### ISSUE TITLE
StandardNodeMaterial's roughness does not work with environment from ReflectorNode

### ISSUE THREAD

**Issue by zhutq** [mrdoob/three.js#14722]
##### Description of the problem
I'm trying to add a ground to display some rough reflections of the objects.

I add a mirror ground to examples/webgl_materials_nodes.html, like that in examples/webgl_mirror_nodes.html:
```
var planeGeo = new THREE.PlaneBufferGeometry( 100.1, 100.1 );

// reflector/mirror plane
var geometry = new THREE.PlaneBufferGeometry( 100, 100 );
var groundMirror = new THREE.ReflectorRTT( geometry, { clipBias: 0.003, textureWidth: WIDTH, textureHeight: HEIGHT } );

var mirror = new THREE.ReflectorNode( groundMirror );

let groundMirrorMaterial = new THREE.StandardNodeMaterial();
groundMirrorMaterial.environment = mirror;
var roughness = new THREE.FloatNode( 0.2 );
groundMirrorMaterial.roughness = roughness;

var mirrorMesh = new THREE.Mesh( planeGeo, groundMirrorMaterial );
groundMirror.add( mirrorMesh );
groundMirror.rotateX( - Math.PI / 2 );
scene.add( groundMirror );
```
But the reflections are not affected by  `groundMirrorMaterial.roughness`.

With `groundMirrorMaterial.environment = new THREE.CubeTextureNode(cubemap);`, a normal env map:
![screenshot from 2018-08-16 17-07-46](https://user-images.githubusercontent.com/1166360/44199665-470cc700-a177-11e8-8f79-cc269697e6ef.jpg)

With `groundMirrorMaterial.environment = mirror;`, same roughness value:

![screenshot from 2018-08-16 17-08-31](https://user-images.githubusercontent.com/1166360/44199672-4bd17b00-a177-11e8-9fba-83ff2de68052.jpg)

Maybe it's similiar to #14709 .

##### Three.js version

- [x] Dev
- [ ] r95
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by sunag** [comment]
It is an old issue in sea3d forum: 

I will have to check later, but I think it is because `ReflectorRTT` `WebGLRenderTarget` is not a `Power of 2` texture, maybe some other reason? 
`roughness` need `mipmaps` to blur effect.

**Comment by sunag** [comment]
> Maybe it's similiar to #14709 .

In fact that for texture instance of cube is need others BRDF calcs.

**Comment by sunag** [comment]
@zhutq You use `WebGLRenderTargetCube` or a CubeMap texture in environment?
I already got it blur the `WebGLRenderTarget` but not physical based.

![image](https://user-images.githubusercontent.com/502810/44278453-631a7200-a224-11e8-8fc0-dd594f01d0b8.png)

**Comment by zhutq** [comment]
@sunag Sorry I didn't quite understand the question.

If I use a `CubeTextureNode` with loaded env images for the floor's `environment` there is blurry.
But if I plug `ReflectorNode` into `environment` there's no blurry.

I don't know where to use `WebGLRenderTargetCube` or `CubeMap`?
Do I need to rely on `BlurNode` instead of `roughness` to introduce blurry (which is not pbr)?

By the way, here's what I've got right now (same as the images in the prev post; sorry didn't get it work in jsfiddle due to imports):
https://cdn.rawgit.com/zhutq/filehost/a95ae2f9/three.js/webgl_materials_nodes.html

Thank you!

**Comment by sunag** [comment]
> @sunag Sorry I didn't quite understand the question.

Sorry too do not make me understand.

I already make `ReflectorNode` work with `roughness` which is the basis of issue, but the quality of `blur` is bad, `PMREM` for example create a good environment `irradiance cube` what is not happening here and maybe I will have to modify the shader if `MaxMipLevel` is not work with planar reflection like (see in `roughness = 0`  still remains slightly blurred). `BlurNode` could be a alternative to avoid mipmaps to blur but I think it is going to need many steps to irradiance, would not be good.

Code work but is not ready.
https://rawgit.com/sunag/three.js/dev-mirror-roughness/examples/webgl_mirror_nodes.html

In short, it is possible fix this issue for now with this code but this will require more work to obtain good quality.

**Comment by Mugen87** [comment]
Closing since this issue is related to the old node material which has been removed with `r138`.
